lib/utils/npm-utils.js

```diff
@@ -39,20 +39,45 @@ function findPackageJson(startDir) {
     return null;
 }
 
+/**
```

Comment:
We can write this as a function(possible `isRootWorkspace()`), something like:

```js
function isRootWorkspace(packageJsonPath){
  const pkg = JSON.parse(...)
  if(pkg.workspaces !== void 0) return true;
  return fs.existsSync(path.join(packageJsonPath, "./pnpm-workspace.yaml");
}
```

it can be defined in the constructor, so you can just use it when you need it:
```js
this.useWorkspace = isRootWorkspace();
```